valhalla/sif/costconstants.h

```diff
@@ -20,6 +20,8 @@ enum class TravelMode : uint8_t {
   kPublicTransit = 3,
   kMaxTravelMode = 4
 };
+// To avoid collisions with pbf version
+using travel_mode_t = TravelMode;
```

Comment:
in `thor` we make use of `sif::TravelMode` a lot but since i moved the common stuff in proto up to the valhalla namespace we have to tell the code in thor its refering to the sif one not the proto one. to make this easier i made a type alias. probably we should get rid of the non protoversions of some of this stuff too to be honest but ill focus on one thing at a time :smile: